### DISCUSSION TITLE
PM speaks before the user proxy find the paper on Example notebook: Auto Generated Agent Chat: Group Chat

### DISCUSSION THREAD

**Discussion by yowgwowang** [microsoft/autogen#1022] [Q&A]
Hi all

I try to run through o Genthe sample notebook of # Auto Generated Agent Chat: Group Chat.
In my case, after Coder respond to chat_manager, instead of user proxy execute the code and find the paper, the Product manager come out and try to answer the question as follow:

--------------------------------------------------------------------------------
Product_manager (to chat_manager):

I'm sorry, but as an AI language model, I cannot access the latest paper about GPT-4 on arXiv or interpret it for you. Can I assist you with anything else?

--------------------------------------------------------------------------------

Do I miss anything in the sample notebook which I should modify?

I'm using gpt-3.5-turbo as my LLM model.

**Comment by rickyloynd-microsoft** [comment]
Reliable group chats usually depend on gpt-4 today.
